src/sentry/api/endpoints/organization_has_mobile_app_events.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+
+from django.core.cache import cache
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.snuba import discover
+
+# one day cache
+CACHE_TTL = 24 * 60 * 60
+
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationHasMobileAppEvents(OrganizationEventsEndpointBase):
+    def get(self, request, organization):
+        # cache is unique to an org
+        cache_key = f"check_mobile_app_events:{organization.id}"
```

Comment:
Should the cache key vary by projects?

Reply:
@markstory Nah, because we are querying all projects for a given organization